nodejs/awsx/elasticloadbalancingv2/listenerRule.ts

```diff
@@ -33,9 +33,11 @@ export class ListenerRule extends pulumi.ComponentResource {
 
     constructor(name: string, listener: x.elasticloadbalancingv2.Listener,
                 args: ListenerRuleArgs, opts: pulumi.ComponentResourceOptions = {}) {
-        super("awsx:x:elasticloadbalancingv2", name, {}, { parent: listener, ...opts });
+        // We forgot to add the `ListenerRule` part of the name.  Add it in and create an alias from
+        // the previous incorrect name.
+        const aliases = [pulumi.createUrn(name, "awsx:x:elasticloadbalancingv2", opts.parent)];
```

Comment:
I think this can (and should to be correct here - since the wrong parent happens to being passed here) be:

```ts
const aliases = [{ type: "awsx:x:elasticloadbalancingv2"})];
```

Reply:
you're right.  will fix.